### ISSUE TITLE
Correlated or not?

### ISSUE THREAD

**Issue by ChrisMantegna** [RobertsLab/resources#1871]
Here are the p450 biomarker values by site and the SumPAHs by site.

[Here is the link](https://rpubs.com/cmantegna/o8p450plots) to the plots if you don't want to view from the screenshots below.

![p450](https://github.com/ChrisMantegna/WDFWmussels/blob/main/output/Screenshot%202024-04-05%20at%201.43.01%E2%80%AFPM.png?raw=true)
![SumPAHs](https://github.com/ChrisMantegna/WDFWmussels/blob/main/output/Screenshot%202024-04-05%20at%201.42.50%E2%80%AFPM.png?raw=true)

**Comment by sr320** [comment]
Not correlated - blue is PAH..

https://d.pr/i/Es4WGG

**Comment by ChrisMantegna** [comment]
Great. that means all of that work showed no real correlation. 
Ok, I was going over the analyte data and this qualifier table. There are several PAH samples that are marked with D for detected and then B. I am going to remove them and see if that makes a difference. It's a chunk of values. If that doesn't make a difference, I'm assuming I just discuss environmental influences, sample handling/ freeze time/ transplanted mussels/ still see expression/potential need for more sensitive assay? 

![qualifier table](https://github.com/ChrisMantegna/WDFWmussels/blob/main/output/Screenshot%202024-04-05%20at%202.07.11%E2%80%AFPM.png?raw=true)

**Comment by sr320** [comment]
to me, you would want to check each analyte separately?

**Comment by ChrisMantegna** [comment]
ok. that's where i'll start.

**Comment by ChrisMantegna** [comment]
there is no correlation or pattern. the SOD biomarker is looking better right now.

**Comment by ChrisMantegna** [comment]
I ran individual analyte plots by site and plots of p450 against the analyte to see if anything shows. Of course I cannot get them to save to an outputs folder so I added the data sheets to [Google Drive Folder](https://drive.google.com/drive/folders/117-nyD_15vDRNMBNEIAgbrxVdImvu9vz?usp=drive_link) and the code is [09-datacleaning](https://github.com/ChrisMantegna/WDFWmussels/blob/main/code/09-datacleaning.Rmd) if you want to double check. I merged my data frames carefully to maintain all individual mussels, I only used detected analytes. I'm at a loss for what to do with the p450 data. The dataframe in the google folder is alldata.csv.
